sources/web/datalab/static/datalab.js

```diff
@@ -100,15 +104,13 @@
     WebSocketShim.CLOSED = 0;
     WebSocketShim.OPEN = 1;
 
-    var nativeWebSocket = window.WebSocket;
-    window.WebSocket = WebSocketShim;
+    if (shouldShimWebsockets()) {
+        debug.log('Replacing native websockets with socket.io');
+        var nativeWebSocket = window.WebSocket;
```

Comment:
This creates a local var, that won't be accessible outside. Should window.nativeWebSocket be set instead?